Bottom: 06d24eb3466a74a466c85da62dafd0e2a290ec66
Top:    b5459e9c317707c0792bf02c210bafaa4a4031e9
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-08-07 14:33:29 +0100

Refresh of generate-pcb-menu-res-and-gpcb

---

diff --git a/src/gpcb-menu.res b/src/gpcb-menu.res.in
similarity index 100%
rename from src/gpcb-menu.res
rename to src/gpcb-menu.res.in
diff --git a/src/pcb-menu.res b/src/pcb-menu.res.in
similarity index 100%
rename from src/pcb-menu.res
rename to src/pcb-menu.res.in
